CheckDrive.Api/CheckDrive.Application/Extensions/DependencyInjection.cs

```diff
@@ -1,24 +1,32 @@
-﻿using CheckDrive.Application.Interfaces;
+﻿using CheckDrive.Application.Interfaces.Authorization;
 using CheckDrive.Application.Interfaces.Review;
+using CheckDrive.Application.Services.Authorization;
+﻿using CheckDrive.Application.Interfaces;
 using CheckDrive.Application.Services;
 using CheckDrive.Application.Services.Review;
+using CheckDrive.Domain.Authorization;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace CheckDrive.Application.Extensions;
 
 public static class DependencyInjection
 {
-    public static IServiceCollection RegisterApplication(this IServiceCollection services)
+    public static IServiceCollection RegisterApplication(
+        this IServiceCollection services,
+        IConfiguration configuration)
```

Comment:
if parameter names are too long, or more than 2 you can put them on a separate line from method declaration, otherwise keep them on the same line.